packages/golang/src-os-pidfd_linux.go.patch

```diff
@@ -0,0 +1,11 @@
+--- a/src/os/pidfd_linux.go
++++ b/src/os/pidfd_linux.go
+@@ -121,7 +121,7 @@
+ }
+ 
+ func pidfdWorks() bool {
+-	return checkPidfdOnce() == nil
```

Comment:
This patch only affect the binary compiled with golang in Termux, and will not work if the golang binary is cross-compiled.

This should use the same way as `/etc/resolv.conf`, see https://github.com/termux/termux-packages/blob/master/packages/golang/fix-hardcoded-etc-resolv-conf.sh

Reply:
Thank you for the review. Sadly, I'm not familiar with golang. Please feel free to create a separate pull request with my patch.